prow/pjutil/pjutil.go

```diff
@@ -27,8 +29,30 @@ import (
 	"k8s.io/test-infra/prow/kube"
 )
 
+const (
+	jobNameLabel = "prow.k8s.io/spec.name"
+	jobTypeLabel = "prow.k8s.io/spec.type"
+	orgLabel     = "prow.k8s.io/spec.refs.org"
+	repoLabel    = "prow.k8s.io/spec.refs.repo"
+	pullLabel    = "prow.k8s.io/spec.refs.pull"
```

Comment:
do they need to be this long? I'd expect `org` to be a safe label on prowjobs and be a lot quicker to type and easier to remember

Reply:
I thought it was best practice to namespace labels? The only non namespaced label we have today has a TO-DO to namespace it. I'm fine either way

Reply:
Hmm yeah automatically applied labels are generally expected to be namespaced..
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set
We can probably cut down to just "prow.k8s.io/org" though?